Caveat this only works for IdP-Initiated SSO (#90)
mulesoft/docs-access-management: modules/ROOT/pages/saml-bearer-token.adoc

This topic describes how to obtain the bearer token from Anypoint Platform. If you are using an identity provider configured to use SAML 2.0, federated users within your organization can access platform APIs using the bearer token.

> +

Reviewer: ```suggestion
```

[NOTE]

> This method works for IdP-Initiated SSO only as SP-Initiated SSO does not support unsolicited assertions. 

Reviewer: ```suggestion
Obtaining an API bearer token using a SAML assertion works only for IdP-initiated single sign-on. Service provider-initiated SSO does not support unsolicited assertions. 
```

> +

Reviewer: ```suggestion
```

+
[NOTE]
The SAML response must be base64 encoded. 

> If the SAML response was received from a HAR file via a text editor, the string is URL encoded. Please use a URL decoder such as urldecoder.org to decode the string first and try again.

Reviewer: ```suggestion
If the SAML response string is URL encoded, you must decode the string before running the `curl` command.
```
